Source/WebCore/testing/Internals.cpp

```diff
@@ -1643,7 +1644,10 @@ void Internals::setCanShowPlaceholder(Element& element, bool canShowPlaceholder)
 
 Element* Internals::insertTextPlaceholder(int width, int height)
 {
-    return frame()->editor().insertTextPlaceholder(IntSize { width, height }).get();
+    auto* localFrame = frame();
```

Comment:
I’d think this would need to be `RefPtr` instead of `auto*` in our new safer C++ model.

Not sure why this is using an `if` statement while the others are using `? :` instead.

Reply:
changed to `? :` style and to use protectedFrame().